dashboard/config/blocks/Dancelab/Dancelab_ai.json

```diff
@@ -11,7 +11,7 @@
         "dummy": true,
         "inline": true,
         "customOptions": {
-          "assetUrl": "media/ai-bot-border.png",
+          "assetUrl": "media/ai-bot-mini-border.svg",
```

Comment:
I opted to use .svg file, but unsure if I should go back to the .png file instead for consistency.

Reply:
Confirmed offline we definitely want to use .svg file.